src/IssueViz.Security/IssueVizSecurityPackage.cs

```diff
@@ -34,7 +33,7 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security
     [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [Guid("D7D54E08-45E1-49A6-AA53-AF1CFAA6EBDC")]
     [ProvideMenuResource("Menus.ctmenu", 1)]
-    [ProvideToolWindow(typeof(HotspotsToolWindow), MultiInstances = false, Style = VsDockStyle.Tabbed, Window = ToolWindowGuids.SolutionExplorer, Width = 325, Height = 400)]
+    [ProvideToolWindow(typeof(HotspotsToolWindow), MultiInstances = false, Style = VsDockStyle.Float, Width = 700, Height = 250)]
```

Comment:
Why floating? Is it possible to dock it to the error list by default?

Reply:
Yep, a floating control can still be docked. I'll experiment with docking it into the error list.